vendor/src/github.com/docker/libcontainer/nsinit/pause.go

```diff
@@ -39,7 +39,7 @@ func toggle(state cgroups.FreezerState) error {
 		return err
 	}
 
-	if systemd.UseSystemd() {
+	if container.Cgroups.Driver == "systemd" && systemd.UseSystemd() {
```

Comment:
much `container.Cgroups.Driver == "systemd"`. Is there a reason why it shouldn't just be `container.Cgroups.IsSystemd()`?